src/Release/Release.php

```diff
@@ -8,8 +8,9 @@
 
 
 use DateTime;
+use Clearbooks\Labs\Release\Entity\PublicRelease;
 
-class Release
+class Release implements PublicRelease
```

Comment:
There would be nothing against multiply inheriting both the Entity interface, and the Usecase boundary interface in this single concrete implementation.

Perhaps the namespace would be misleading then.

Perhaps just ```Clearbooks\Labs\Release``` namespace would suffice.

# Concrete
```php
class Release implements Entity\PublicRelease, UseCase\GetPublicRelease\PublicRelease
```

# UseCase boundary
```php
namespace Clearbooks\Labs\Release\UseCase\GetPublicRelease;

interface PublicRelease extends Clearbooks\Labs\Release\Common\PublicRelease
```

# Entity interface

```php
namespace Clearbooks\Labs\Release\Entity;

interface PublicRelease extends Clearbooks\Labs\Release\Common\PublicRelease
```

# Abstract DRY definition

```php
namespace Clearbooks\Labs\Release\Common;

interface PublicRelease
{
    /**
     * @return boolean
     */
    public function isVisible();
    /**
     * @return \DateTimeInterface
     */
    public function getReleaseDate();
    /**
     * @param $visivilityStatus
     */
    public function setVisible( $visivilityStatus );
}
```